src/styles/blocks/product-section.scss

```diff
@@ -0,0 +1,32 @@
+$column-count: 1;
+
+.product-section {
+  display: grid;
+  justify-content: center;
+
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat($column-count, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    $column-count: 2;
+    & {
+      grid-template-columns: repeat($column-count, 200px);
```

Comment:
To use a variable like this doesn't make much sense. You can use `css` custom properties instead.

```scss
.product-section {
  --column-count: 1;

  grid-template-columns: repeat(var(--column-count), 200px);
  
  @media (min-width: 488px) {
    --column-count: 2;
  }
}
```
Something like this